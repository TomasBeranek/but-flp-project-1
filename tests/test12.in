K,L,M
k,l,m
L
K->#
K->kkK
L->kK
L->lL
L->llL
L->mM
M->#
M->mmM
