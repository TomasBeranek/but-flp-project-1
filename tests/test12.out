0,1,2,3,4,5
klm
4
3,5
0,m,5
1,l,4
2,k,3
3,k,2
4,k,3
4,l,1
4,l,4
4,m,5
5,m,0
